src/engine/drpc_ras.c

```diff
@@ -325,7 +312,7 @@ ds_notify_pool_svc_update(uuid_t *pool, d_rank_list_t *svcl)
 		       RAS_TYPE_STATE_CHANGE, RAS_SEV_INFO, NULL /* hwid */,
 		       &rank /* rank */, NULL /* jobid */, pool,
 		       NULL /* cont */, NULL /* objid */, NULL /* ctlop */,
-		       &evt);
+		       &evt, true /* wait_for_resp */);
```

Comment:
Agreed, that's what earlier versions of this PR assume, until I recalled recently that ds_notify_pool_svc_update may need to know if the event is received by the MS successfully. If not, the PS may want to retry until the MS receives the event, so that the pool service rank list on the MS is up to date. (I had some uneasiness when seeing us using RAS events to do this update.) I'll keep this option.

Reply:
Works for me. Thanks for the explanation.